internal/database/locker/locker_test.go

```diff
@@ -110,3 +110,27 @@ func TestLockBlockingAcquire(t *testing.T) {
 		t.Errorf("lock not acquired before release")
 	}
 }
+
+func TestLockBadTransactionState(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	db := dbtesting.GetDB(t)
```

Comment:
Just tryna advocate for using the "new way" ™️, feel free to ignore

```suggestion
	db := dbtest.NewDB(t, "")
```